profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry.install

```diff
@@ -85,18 +85,96 @@ function tmgmt_poetry_schema() {
 }
 
 /**
+ * Implements hook_requirements().
+ */
+function tmgmt_poetry_requirements($phase) {
```

Comment:
We discussed that maybe requirements part should be moved to the 'nexteuropa_dgt_connector' feature .install file. In that way we could unblock enabling the tmgmt_poetry module for the development purposes in future.